src/xunit.performance.metrics/BenchmarkDurationEvaluator.cs

```diff
@@ -0,0 +1,30 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
```

Comment:
I think if it's easy to undo, then yes, since it's not complete.  Then go ahead and merge.